packages/core/src/Scene.ts

```diff
@@ -263,6 +266,13 @@ export class Scene extends EngineObject {
     }
   }
 
+  /**
+   * Post Process manager.
+   */
+  get postProcessManager(): PostProcessManager {
+    return this._postProcessManager;
+  }
```

Comment:
??? Why public API?

Can you guarantee that the next version will not be modified?

Reply:
done